modules/apps/headless-apio/structure/structure-apio-api/src/main/java/com/liferay/structure/apio/architect/util/StructureFieldConverter.java

```diff
@@ -31,6 +31,16 @@
 	 */
 	public String getFieldDataType(String dataType);
 
+	/**
+	 * Gets the field data type.
+	 *
+	 * @param  dataType the {@code DDMFormField#dataType}
+	 * @param  type the {@code DDMFormField#type}
+	 * @return the field data type
+	 * @review
+	 */
+	public String getFieldDataType(String dataType, String type);
```

Comment:
You can add a default implementation to avoid a major change in semantic versioning:
```suggestion
public default String getFieldDataType(String dataType, String type) {
return getFieldDataType(dataType);
}
```